src/components/StaticContent/CoreConcepts.res

```diff
@@ -1,11 +1,27 @@
 let gorillaOnVine = "/img/wildcardsimages/gorilla-on-vine.png"
 
+/*
 open Globals
 open Rimble
 
+// //  When (if) you want to add this component back with a muggle toggle add this to the page
+      // crypto: {
+      //   nft: "Non-Fungible Token",
+      //   whatIsANFT:
+      //     " is a way to ensure digital assets are unique and easily tradable on a blockchain.",
+      //   harbergerTax: "Harberger Tax"
+      // },
+      // muggle: {
+      //   nft: "Blockchain Animal",
+      //   whatIsANFT:
+      //     " is a digital animal that is completely unique and easily tradable on a blockchain.",
+      //   harbergerTax: "Monthly Pledge"
+      // }
```

Comment:
This will be useful when we add back this component as a new page.

I think it should be a page that you go to from the NAV bar saying "What is this?" or "How it works" or "FAQ"